modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -7,32 +7,66 @@
 
-Currently, Data Mover does not support Google Cloud Storage (GCS) buckets.
-=====
+In OADP 1.2.x Data Mover VolumeSnapshotBackups (VSBs) and VolumeSnapshotRestores (VSRs) are queued using the VolumeSnapshotMover (VSM). The VSM's performance is improved by specifying a concurrent number of VSBs and VSRs simultaneously in Progress. After all async plugin operations are completed, the backup is marked as completed.
+
+
+[NOTE]
+====
+The OADP 1.2.x Data Mover is a fully supported Red Hat feature.
+
+The OADP 1.1.x Data Mover is a Technology Preview feature.
+====
+:FeatureName: The OADP 1.1.x Data Mover
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[NOTE]
+====
+Red Hat recommends that customers who use OADP 1.2.x Data Mover to back up and restore ODF CephFS volumes, upgrade or install OCP 4.12 for improved performance. OADP Data Mover can leverage CephFS shallow volumes in OCP 4.12+, which based on our testing improves the performance of backup times.
+
+* https://https://issues.redhat.com/browse/RHSTOR-4287[CephFS ROX details]
+* https://https://github.com/ceph/ceph-csi/blob/devel/docs/cephfs-snapshot-backed-volumes.md[Provisioning and mounting CephFS snapshot-backed volumes]
+
+For more information about OADP 1.2.x with CephS [name of topic], see ___.
+====
 
 .Prerequisites
 
-* You have verified that the `StorageClass` and `VolumeSnapshotClass` custom resources (CRs) support CSI.
+* You have verified that the `StorageClass` and `VolumeSnapshotClass` Custom Resources (CRs) support CSI.
 
 * You have verified that only one `volumeSnapshotClass` CR has the annotation `snapshot.storage.kubernetes.io/is-default-class: true`.
++
+[NOTE]
+====
+In OCP 4.12+ verify that this is the only default `volumeSnapshotClass`.
+====
+
+* You have verified that `volumeSnapshotClass` CR `deletionPolicy` is set to `Retain`.
 
 * You have verified that only one `storageClass` CR has the annotation `storageclass.kubernetes.io/is-default-class: true`.
 
 * You have included the label `{velero-domain}/csi-volumesnapshot-class: 'true'` in your `VolumeSnapshotClass` CR.
 
+* You have verified that the `OADP namespace` has the annotation `oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='true'`.
++
+[NOTE]
+====
+In OADP 1.1.x the above setting is mandatory.
+
+In OADP 1.2.x the privileged-movers setting is not required in most scenarios. The restoring container permissions should be adequate for the Volsync copy. In some user scenarios, there may be permission errors that the `privileged-mover`= `true` setting should resolve.
+====
+
 * You have installed the VolSync Operator by using the Operator Lifecycle Manager (OLM).
 +
 [NOTE]
 ====
-The VolSync Operator is required only for use with the Technology Preview Data Mover. The Operator is not required for using OADP production features.
+The VolSync Operator is required for using OADP Data Mover feature.
```

Comment:
Either "...for using the...feature." or "...for using "OADP Data Mover." -- whichever fits your usage in this section.